niedoida/src/do_td.cpp

```diff
@@ -41,14 +47,28 @@ namespace {
                 "energies (eV)",
                 arma::vec(misc::hartree_to_ev * rtd.energies()));
 
+            std::size_t a_max {0};
+            std::size_t i_max {0};
+            double max_v {0};
```

Comment:
in such cases initialization syntax like `std::size_t a_max = 0;` is preferred, please correct.

Reply:
please fix the third one as well